frontend/src/locales/zh-CN.yml

```diff
@@ -27,3 +35,54 @@ kbar:
   preferences:
     common: 偏好设置
     change-language: 更改语言…
+bbkit:
+  alert:
+    ok: 确认
+    cancel: 取消
+settings:
+  sidebar:
+    account: 账户
+    profile: 配置
+    workspace: 工作区
+    general: 通用
+    members: 成员
+    version-control: 版本控制
+    plans: 价格方案
+  profile:
+    email: 邮箱
+    role: 角色
+    password: 密码
+    password-placeholder: 数据敏感 - 仅写入
+    password-confirm: 确认密码
+    password-confirm-placeholder: 再次输入以确认
+    password-mismatch: 密码输入不一致
+    plan: （升级到团队方案来开启角色管理）
+  workspace:
+    url-section: SQL Console URL
+    url-tip: "提示: 用 %s 作为真实数据库名称的占位符"
+    tip: 如果你的团队使用一个单独的 SQL 控制台，如 phpMyAdmin，那么可以在这里提供它的 URL 模板，Bytebase 就可以将控制台链接到相关的数据库和表的界面上。
+    tip-link: 详细指引
+  members:
+    active: 已激活
+    inactive: 未激活
```

Comment:
Not sure is good for `active` & `inactive` translation

Reply:
SGTM